pkg/governance-scripts/contracts/TimelockAuthorizerTransitionMigrator.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-interfaces/contracts/vault/IBasicAuthorizer.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Address.sol";
+import "@balancer-labs/v2-vault/contracts/authorizer/TimelockAuthorizer.sol";
+
+/**
+ * @notice Migrates permissions granted after `TimelockAuthorizer` deployment from the old authorizer.
+ */
+contract TimelockAuthorizerTransitionMigrator {
+    using Address for address;
+
+    TimelockAuthorizer public immutable timelockAuthorizer;
+    RoleData[] public rolesData;
+
+    struct RoleData {
+        address grantee;
+        bytes32 role;
+        address target;
+    }
+
+    bool private _migrationDone;
+
+    /**
+     * @dev Reverts if rolesData contains a role for an account which doesn't hold the same role on the old Authorizer.
+     */
+    constructor(
+        IBasicAuthorizer oldAuthorizer,
```

Comment:
Could even hard-code this. Or the script could pull it in from the Authorizer deployment task

Reply:
That's right; I'm pulling the address from the task in the deployment script and fork test in #2151.